sdk/cosmosdb/cosmos/review/cosmos.api.md

```diff
@@ -168,6 +168,7 @@ export const Constants: {
         CollectionPartitionInfo: string;
         CollectionServiceInfo: string;
         RetryAfterInMilliseconds: string;
```

Comment:
Is having`RetryAfterInMilliseconds` and `RetryAfterInMs` intended?

Reply:
Oh I see, is it that we want to keep it but mark it as deprecated?

Reply:
The concern was if I removed the old one, we'd have a breaking change on our hands.